Выполнено задание 1

valerykoshkin opened the pull request.

valerykoshkin committed:
add pom.xml

valerykoshkin committed:
Выполнено задание 1

pomegranate282922 commented on src/test/java/BunTest.java:
```diff
@@ -0,0 +1,43 @@
+    public void checkGetPrice() {
+        Bun bun = new Bun(name, price);
+        float actualPrice = bun.getPrice();
+        assertEquals(price, actualPrice, 0);
```
Дельту лучше вынести как константа и переиспользовать

valerykoshkin replied:
Вынес константу в интерфейс Constants

pomegranate282922 commented on src/test/java/BurgerTest.java:
```diff
@@ -0,0 +1,95 @@
+    }
+
+    @Test
+    public void CheckRemoveIngredient() {
```
Методы именуем начиная со строчной буквы 
Нужно исправить везде

valerykoshkin replied:
Поправил

pomegranate282922 commented on src/test/java/BunTest.java:
```diff
@@ -0,0 +1,43 @@
+
+    @Parameterized.Parameters
+    public static Object[][] bun() {
+        return new Object[][]{
```
Отлично что используются разные классы эквивалентности

pomegranate282922 reviewed (commented):
Хорошая работа! Применена параметризация, уместно используются Моки и стабы. 
замечания:
1. Нет тестов на enum 
2. Нейминг

valerykoshkin committed:
Правки по ревью первого задания
-Добавлен параметризованный тест на enum
-Поправлен нейминг
-Вынесена константа дельты в интерфейс

valerykoshkin commented:
Добавил тест на enum, за основу взял: https://stackoverflow.com/questions/32150174/how-to-test-enums-using-junit

pomegranate282922 reviewed (approved):
Отлично, зачет!
